### PR TITLE
Centralize verification if value is a object callable.

### PR DESCRIPTION
**Pull request by marcusesa**

### CODE REVIEW

File: src/Pimple/Container.php [hunk 1]
```diff
@@ -146,6 +146,18 @@ public function offsetUnset($id)
     }
 
     /**
+     * Check if $value is a object callable.
+     *
+     * @param mixed $value
```
Comment (stof) [inline src/Pimple/Container.php]: technically, this method always receives a callable

**Comment by stof** [comment]
Private methods should be declared after public ones

**Comment by marcusesa** [comment]
> technically, this method always receives a callable
Yes but how we can receive this value through of public interfaces without type hinting, we are not sure about type.

**Comment by stof** [comment]
@marcusesa the public methods calling it are already expecting a callable (and they would be typehinted in PHP 5.4+ but Pimple supports 5.3)

**Comment by marcusesa** [comment]
Yes @stof, because Pimple supports PHP 5.3 and we cant use type hinting, we have the expectation and not sure. So we should use in doc a expectation?

**Comment by stof** [comment]
@marcusesa we already rely on the phpdoc to define the signature of the class expecting a callable.

**Comment by marcusesa** [comment]
Ok @stof.

**Comment by fabpot** [comment]
I prefer the current way as it saves one method call. The Pimple code does not change that much and having such redundancies is not really a problem.

**Pull request closed without merging**